stories/Page.stories.tsx

```diff
@@ -1,21 +1,15 @@
 import { Story, Meta } from '@storybook/react'
```

Comment:
This is a new file, its just reusing the name space for our actual page, instead of the example Page (now Page_example)